examples/k8s/influx.yml

```diff
@@ -0,0 +1,145 @@
+#Create InfluxDB
+--- 
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: influx
+  namespace: monitoring
+  labels:
+    app: unifi-poller
+    type: influx
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: unifi-poller
+      type: influx
+  template:
+    metadata:
+      labels:
+        app: unifi-poller
+        type: influx
+    spec:
+      containers:
+      - name: influxdb
+        image: influxdb:latest
```

Comment:
`influxdb:latest` is incompatible. It should be `influxdb:1.8`.

Reply:
thanks for the info... i changed the version